### PR TITLE
[DOC] Add Collaborators page

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #374](https://github.com/yaooqinn/kyuubi/pull/374)    Target Issue Test Plan [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
Will the doc auto build and deploy when PR merged? Or I should do additional actions?

**Comment by yaooqinn** [comment]
yes but plz follow this guide https://kyuubi.readthedocs.io/en/latest/tools/build_document.html to verify locally first

File: docs/community/collaborators.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<script async defer src="https://buttons.github.io/buttons.js"></script>
```
Comment (yaooqinn) [inline docs/community/collaborators.md:1]: This is not needed

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.1.0
